assets/js/blocks/product-query/inspector-controls.tsx

```diff
@@ -191,6 +192,13 @@ export const withProductQueryControls =
 						) }
 					</ToolsPanel>
 				</InspectorControls>
+				{
+					// Hacky temporary solution to display the feedback prompt
+					// at the bottom of the inspector controls
+				 }
+				<InspectorControls __experimentalGroup="color">
+					<ProductQueryFeedbackPrompt />
+				</InspectorControls>
```

Comment:
Given I anticipate this feedback prompt not being around for a while I think this is sufficient for now. When we release the block to WooCommerce core, we'll have to test behaviour against versions of WP that WC Core supports.